### ISSUE TITLE
fetchxml with top= tag

### ISSUE THREAD

**Issue by pyro-mm** [AleksandrRogov/DynamicsWebApi#98]
When adding the top tag to the fetchxml string <fetch top="1" > I always get the error
"Error: The top attribute can't be specified with paging attribute page"

I just added to line 875
if (pageNumber < 0) replacementString = '$1 ';
When I call executeFetchXml(entityname, sFetchXML, null, -1, null) it seems to work.

**Comment by AleksandrRogov** [comment]
hi @pyro-mm , I remember I had to include a paging number in every fetch request because I wanted to retrieve a paging cookie and for some reason it was not doing it back in 8.2 without adding a "paging" attribute to fetch xml, I don't know if it's fixed right now, did not test it.

To be honest, I did not take into account the fact that devs can use "top" inside a fetch xml as well. So, I am going to make some changes and publish a patch asap.

Thanks for submitting the issue!
